x/vbr/legacy/v3.0.0/migrate_test.go

```diff
@@ -0,0 +1,63 @@
+package v3_0_0
+
+import (
+	"reflect"
+	"testing"
+
+	v220vbr "github.com/commercionetwork/commercionetwork/x/vbr/legacy/v2.2.0"
+	"github.com/commercionetwork/commercionetwork/x/vbr/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+var params = types.Params{
```

Comment:
It's ok, but can you check  `types.DefaultParams()`: default params earn value should be  0.5 but the method return 50.